FileProcessor/DownloadedFile.txt

```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ligula mi, posuere at tincidunt in, sodales facilisis odio. Donec consequat tincidunt orci, id pretium nunc malesuada non. Donec ultricies mauris id lobortis sagittis. Donec tincidunt massa non ipsum vestibulum, a auctor est molestie. Nullam rutrum lectus vel fermentum tincidunt. Vivamus id pulvinar lacus. In porta fringilla metus, quis ultrices erat tempor id. Fusce tellus mauris, dapibus et molestie sed, ornare ut velit. Vestibulum nunc nisi, tincidunt tristique mollis in, dignissim vel purus. Sed pellentesque porttitor mauris in rutrum. Ut vitae tristique tellus, id gravida sem. Aliquam eu lorem at est pretium eleifend. Nulla a interdum lacus, ut eleifend diam. Suspendisse tincidunt, nulla sed condimentum tincidunt, nisl ipsum dignissim lorem, a pellentesque dolor lacus pharetra lectus.
```

Comment:
What is this Lorem ipsum for?

Reply:
This is the file that was downloaded in the sample program using the laserfiche api.